tools/aa/commands/InitCommand.java

```diff
@@ -95,11 +97,41 @@ public boolean run(String[] args) {
     return true;
   }
 
+  private boolean processArgs(String[] args) {
+    boolean isAaSetUp = args[0].equals("aa");
```

Comment:
Sorry, I think this actually is better:
s/isAaSetUp/isAaAlreadySetUp

Reply:
- [x] Done